src/CSharpMongoMigrations/Extensions/BsonDocumentExtensions.cs

```diff
@@ -69,6 +83,37 @@ public static void ChangeValue(this BsonDocument document, string property, obje
 
             document.Remove(property);
             document.Add(@new);
-        }        
+        }
+
+        /// <summary>
+        /// Change the property value.
+        /// </summary>
+        /// <param name="document">The target document.</param>
+        /// <param name="property">The desired property name.</param>
+        /// <param name="value"></param>
+        public static void ChangeValue(this BsonDocument document, string property, Guid value)
```

Comment:
The same as above. Could you please just invoke the polymorphic method? I believe it will be better.
And, please add the missing unit tests for the new extension methods.

Reply:
Agree, the unit test are ready in the latest commit